lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
> Is it a real problem?

On the small code-base no, on the large one, considering cumulative impact with other patches... might be, if you look on user perception - http://www.sitepoint.com/comparison-javascript-linting-tools#jscs

> Slowest of the four, but this is not a problem in typical use

Which might be a not fair conclusion, but a thing we should keep an eye on. "Slowest" - don't want to be categorized like that